torch/_inductor/ir.py

```diff
@@ -4399,7 +4399,11 @@ def __init__(self, x, indices, values, accumulate):
         )
         self.name = V.graph.register_buffer(self)
         self.python_kernel_name = "aten.index_put_"
-        self.cpp_kernel_name = "at::index_put_out"
+        self.cpp_kernel_name = (
+            "aoti_torch_index_put_out"
+            if config.aot_inductor.abi_compatible
+            else "at::index_put_out"
+        )
```

Comment:
Did we add `aoti_torch_index_put_out` in a prior PR and forgot to add this?

Reply:
No, I did a little refactoring from `kernel = kernel.replace("at::", "aoti_torch_")` in `wrapper.py`.